core/lib/makeswift/components/slideshow/slideshow.makeswift.tsx

```diff
@@ -0,0 +1,106 @@
+import {
+  Checkbox,
+  Image,
+  Link,
+  List,
+  Number,
+  Select,
+  Shape,
+  Style,
+  TextArea,
+  TextInput,
+} from '@makeswift/runtime/controls';
+
+import { Slideshow } from '@/vibes/soul/sections/slideshow';
+import { runtime } from '~/lib/makeswift/runtime';
+
+interface Slide {
+  title: string;
+  description: string;
+  showDescription: boolean;
+  imageSrc?: string;
+  imageAlt: string;
+  showButton: boolean;
+  buttonLink?: { href?: string; target?: string };
+  buttonText: string;
+  buttonColor: 'primary' | 'secondary' | 'tertiary' | 'ghost';
+}
+
+interface MSAccordionsProps {
+  className: string;
+  slides: Slide[];
+  autoplay: boolean;
+  interval: number;
+}
+
+runtime.registerComponent(
+  function MSSlideshow({ className, slides, autoplay, interval }: MSAccordionsProps) {
+    return (
+      <Slideshow
+        className={className}
+        interval={interval * 1000}
+        playOnInit={autoplay}
+        slides={slides.map(
+          ({
+            title,
+            description,
+            showDescription,
+            imageSrc,
+            imageAlt,
+            showButton,
+            buttonLink,
+            buttonText,
+            buttonColor,
+          }) => {
+            return {
+              title,
+              description,
+              showDescription,
+              image: { alt: imageAlt, src: imageSrc ?? '' },
```

Comment:
We should never do this (coalescing an image src to `''`). Instead pass `null` for the entire image object if there is no image